<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ccnv</class>
 <widget class="QMainWindow" name="ccnv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>410</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Myriad Pro Light</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Coordinates Converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frmGps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QFrame#frmGps {
border: 1px solid grey;
}*/</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblE_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblE">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>E</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblLat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lat:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGps">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>451</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>GPS Coordinates (WGS 84)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLonDeg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}

QLineEdit:disabled {
color: black;
border: 1px solid grey;
background: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLonDDMM">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}

QLineEdit:disabled {
color: black;
border: 1px solid grey;
background: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLatDDMM">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}

QLineEdit:disabled {
color: black;
border: 1px solid grey;
background: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblN">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblN_2">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>N</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGpsFormat">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>â†”</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLatDeg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}

QLineEdit:disabled {
color: black;
border: 1px solid grey;
background: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblLon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lon:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblDDMMm">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: grey;
}</string>
     </property>
     <property name="text">
      <string>DDDMM.mmm</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblDDddd">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: grey;
}</string>
     </property>
     <property name="text">
      <string>DDD.ddddd</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnZoom">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
    </widget>
    <widget class="QLabel" name="lblZoom">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: blue;
}</string>
     </property>
     <property name="text">
      <string>Zoom Level:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmPx">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro Light</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblPixel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pixel Coordinates</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblPxY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPxX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTileTL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Px Coor from Tile Origin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTilePxY">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTilePxX">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnPxX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnPxY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnTilePxX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: #f5f5f5;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnTilePxY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: #f5f5f5;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frmTile">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lblTileY">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTileX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblTile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tile Coordinates</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTileStr">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tile String (Z-X-Y)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTileStr">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border: 1px solid grey;
background: #f5f5f5;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnTileX">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spnTileY">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myriad Pro Light</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QSpinBox {
border: 1px solid rgb(122,122,122);
background: white;
color: blue;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnFromGps">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GPS â†’ Px + Tile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFromPx">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Px â†’ GPS + Tile</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFromTile">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tile â†’ GPS + Px</string>
    </property>
   </widget>
   <zorder>frmPx</zorder>
   <zorder>frmGps</zorder>
   <zorder>frmTile</zorder>
   <zorder>btnFromGps</zorder>
   <zorder>btnFromPx</zorder>
   <zorder>btnFromTile</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtLatDDMM</tabstop>
  <tabstop>txtLonDDMM</tabstop>
  <tabstop>btnGpsFormat</tabstop>
  <tabstop>txtLatDeg</tabstop>
  <tabstop>txtLonDeg</tabstop>
  <tabstop>btnFromGps</tabstop>
  <tabstop>btnFromPx</tabstop>
  <tabstop>btnFromTile</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
